modules/gateway/gateway.go

```diff
@@ -81,15 +81,7 @@ import (
 // have previously been outbound peers, as it is less likely that those have
 // been manipulated.
 //
-// TODO: When peers connect to eachother, and when they add nodes to the node
-// list, there is no verification that the peers are running on the same Sia
-// network, something that will be problematic if we set up a large testnet.
-// It's already problematic, and currently only solved by using a different set
-// of bootstrap nodes. If there is any cross-polination (which an attacker
-// could do pretty easily), the gateways will not clean up over time, which
-// will degrade the quality of the flood network as the two networks will
-// continously flood eachother with irrelevant information. Additionally, there
-// is no public key exhcange, so communications cannot be effectively encrypted
+// TODO: There is no public key exhcange, so communications cannot be effectively encrypted
```

Comment:
it's not a super-hard rule -- if there is some code that is very indented, we are more relaxed about wrapping it. However, every comment at the top level (e.g. any docstring on a function, var, const, type) should be wrapped to 80 chars.
Most editors have a way to do this automatically. It's definitely not something you want to do by hand.